test/integration/model.test.js

```diff
@@ -1720,6 +1720,10 @@ describe(Support.getTestDialectTeaser('Model'), () => {
         group: ['data']
       });
       expect(count).to.have.lengthOf(2);
+
+      // The order of count varies across dialects; Hence find element by identified first.
```

Comment:
We could fix this by ordering the to be grouped properties by value but that seems a bit extensive.